### PR TITLE
Clean up of the definitions

### PR DESCRIPTION
**Pull request by Sakurann**
- delete an unused term Claim
- updated definition of a term End User
- added a new term Issuer

### CODE REVIEW

**Review by quartzjer — APPROVED** [review]

File: spec/spec.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Participate:
 
 ## Abstract
 
-The JWT VC Presentation Profile defines a set of requirements against existing specifications to enable the interoperable presentation of [[ref:Verifiable Credentials]] (VCs) between [[ref: Wallets]] and [[ref: Verifiers]].
+The JWT VC Presentation Profile defines a set of requirements against existing specifications to enable the interoperable presentation of [[ref:Verifiable Credentials]] (VCs) between ref: Wallets and ref: Verifiers.
```
Comment (jischr) [inline spec/spec.md:33]: remove these changes? since you update them in another PR and the ref: got weird here.

File: spec/spec.md [hunk 2]
```diff
@@ -95,51 +95,46 @@ This section consolidates in one place common terms used across open standards t
...
 
-[[def:Verifiable Presentation (VP), Verifiable Presentation, VP, Verifiable Presentations]] 
+[[def: Verifiable Presentation (VP)]] 
 ~ A [[ref:Presentation]] that is tamper-evident and has authorship that can be cryptographically verified
 
-[[def:Verifier, Verifiers]]
-~ An entity that receives one or more verifiable credential inside a verifiable presentation for processing. Synonymous
-  with the term [[ref: Relying Party (RP)]]
+[[def: Verifier]]
+~ An entity that receives one or more Verifiable Credential inside a Verifiable Presentation for processing. During presentation of Credentials, Verifier acts as an OAuth 2.0 Client towards the Wallet that is acting as an OAuth 2.0 Authorization Server. The Verifier is a specific case of OAuth 2.0 Client, just like Relying Party (RP) in [@OpenID.Core].
```
Comment (jischr) [inline spec/spec.md:131]:
```suggestion
~ An entity that receives one or more Verifiable Credentials inside a Verifiable Presentation for processing. During presentation of Credentials, Verifier acts as an OAuth 2.0 Client towards the Wallet that is acting as an OAuth 2.0 Authorization Server. The Verifier is a specific case of OAuth 2.0 Client, just like Relying Party (RP) in [[ref: OpenID Connect Core].
```

File: spec/spec.md [hunk 3]
```diff
@@ -95,51 +95,46 @@ This section consolidates in one place common terms used across open standards t
...
 
-[[def:Verification]]
-~ The process in which a [[ref:Verifier]] validates that the verifiable credential inside a verifiable presentation is authentic
-  and a timely statement of the issuer or presenter
+[[def: Issuer]]
+~ An entity that issues Verifiable Credentials.
```
Comment (jischr) [inline spec/spec.md:134]: necessary since we dont discuss issuance?

File: spec/spec.md (hunk 1 above)
Comment (Sakurann) [reply spec/spec.md:33]: i'll take care of this when merging all PRs

File: spec/spec.md (hunk 3 above)
Comment (jischr) [reply spec/spec.md:134]: ah yes good point

**Review by jischr — APPROVED** [review]

**Pull request merged**